scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -38,27 +38,27 @@ def call(final pipelineContext) {
   // for Python, smoke only oldest and latest supported versions
   def SMOKE_STAGES = [
     [
-      stageName: 'Py3.6 Smoke', target: 'test-py-smoke', pythonVersion: '3.6',timeoutValue: 8,
+      stageName: 'Py3.6 Smoke', target: 'test-py-smoke', pythonVersion: '3.6',timeoutValue: 4,
```

Comment:
i would prefer just to extend timeouts that are problematic. Making other timeouts smaller might bring potential problems.